e2e/js_image_oci/README.md

```diff
@@ -24,10 +24,10 @@ Here is what the final image looks like when `platform = "linux/arm64"`:
 app
 |-- main
 |-- main.runfiles
-|   |-- __main__
```

Comment:
See https://github.com/bazelbuild/bazel/issues/24717#issuecomment-2547934581 for the info on `__main__` being removed